2.4/contrib/etc/mongodb.conf

```diff
@@ -0,0 +1,21 @@
+# mongodb.conf
+
+port = 27017
+#fork = true
+pidfilepath = /opt/rh/mongodb24/root/var/run/mongodb/mongodb.pid
+dbpath = /var/lib/mongodb
+
+# Disable data file preallocation.
+noprealloc = true
```

Comment:
good idea, let's queue it up for arch, right after @soltysh 's build notification topic.